src/main/kotlin/org/malachite/estella/people/domain/UserRepository.kt

```diff
@@ -5,6 +5,7 @@ import java.util.*
 
 interface UserRepository {
     fun findAll(): MutableIterable<User>
+    fun findByMail(mail:String): Optional<User>
```

Comment:
Nie wiem, czy nie fajnie byłoby pokombinować z tym, żeby zamiast Optinali były nullowalne wartości w Kotlinie - sam nie znalazłem, ale nie szukałem jakoś bardzo, może akurat coś jest, ale to do przegadania jeszcze

Reply:
Znaczy ja to tak zostawiłem bo
a) Były tak inne
b) Wydaje mi się że jako że ten kod jest generowany w Javie, więc obawiam się że nie obsługuje typów Kotlinowych, ale to ktos musiałby bardziej poczytać

Reply:
Znaczy to bynajmniej nie do tego PRa, ale tak mi się przypomniało. Wydaje mi się, że gdzieś widziałem, że się da, ale chyba było co innego na szybkości.